angular/modules/rxjs/README-ENG.md

```diff
@@ -3,23 +3,51 @@
 ## In this module, students need to learn:
 
 - Principles of reactive programming.
-- `Observable`. `Subject`. `RxJS`  operators.
+- hot/cold `Observable`, `Subject`, `RxJS`  operators.
 
 ## Approximate time to complete this module:
-Theory - 2 hours
-Assignment - 1 hour
 
-## Theory 
-- [Lecture](https://youtu.be/fkmbA1LXAak)
+Theory - 3 hours;
+Assignment - 8 hours
+
+## Theory
+
+- [Former lecture](https://youtu.be/fkmbA1LXAak)
 - [Examples from the lecture](https://github.com/pavelrazuvalau/angular-lectures/tree/master/rxjs-observables)
 - [Slides](https://slides.com/pavelrazuvalau/angular-rxjs)
 
+### Supplements (required):
+
+Following materials also should be considered as learning sources before starting practical tasks
+
+- [`async` pipe in html](https://www.telerik.com/blogs/angular-basics-step-by-step-understanding-async-pipe)
+- [Learn RxJS](https://www.learnrxjs.io)
+- [RxJS API List](https://rxjs-dev.firebaseapp.com/api)
+- [RxJS Crash Cource](https://youtu.be/tGWBy6Vqq9w)
+- [RxJS best practices](https://blog.brecht.io/rxjs-best-practices-in-angular)
+- Hot and Cold Observables:
+    - [Explanation by Predrag Carapic](https://www.decodedfrontend.io/hot-vs-cold-observable-in-rxjs)
+    - [Explanation by Sebastian](https://medium.com/codingthesmartway-com-blog/getting-started-with-rxjs-part-3-hot-and-cold-observables-4713757c9a88)
+- Higher-order Observables
+    - [RxJS documentation](https://rxjs.dev/guide/higher-order-observables)
+
 ## Practical assignment
-- [Angular. RxJS & Observables](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/angular/rxjs-observables-http.md)
+
+- [Angular. RxJS & Observables](../../../tasks/angular/rxjs-observables-http.md)
 - Mentor: Aleh Serhiyenia
 
+### Self-practice
+
+- [Cars and trucks](https://stackblitz.com/edit/angular-rxjs-exercise?file=index.ts)
+- [Fruits game](https://www.rxjs-fruits.com)
+
 ## Additional materials
-- [RxJS API List](https://rxjs-dev.firebaseapp.com/api)
-- [RxJS Marbles](http://rxmarbles.com/)
-- [Learn RxJS](https://www.learnrxjs.io/)
+
 - [Reactive Extensions For JavaScript](https://www.npmjs.com/package/rxjs)
+- RxJS visualization:
+    - [Graphic examples](https://rxjs-visualize.explosionpills.com)
+    - [RxJS Marbles](http://rxmarbles.com/)
```

Comment:
This resource is based on the old Rx version and features outdated syntax. At the same time, I can see it being helpful despite that. Maybe we could add a comment for learners to be aware of the old syntax?

Reply:
Removed this item